image/manifest.go

```diff
@@ -84,7 +85,7 @@ func manifestInstanceFromBlob(src types.ImageSource, manblob []byte, mt string)
 	// need to happen within the ImageSource.
 	case manifest.DockerV2Schema1MediaType, manifest.DockerV2Schema1SignedMediaType, "application/json":
 		return manifestSchema1FromManifest(manblob)
-	case manifest.DockerV2Schema2MediaType:
+	case manifest.DockerV2Schema2MediaType, imgspecv1.MediaTypeImageManifest:
```

Comment:
Is this really sufficient? Looking at `oci_dest.go`, OCI seems to use different MIME types in descriptors, though we pretty much ignore that on the read side, so at least for this PR it does work.

Are Docker schema2 and OCI likely enough to keep in sync in the future that we would continue to use a single implementation, or would it be better to copy&paste&rename right now?

@runcom ?